dds/DCPS/RTPS/ParameterListConverter.cpp

```diff
@@ -576,6 +598,13 @@ bool from_param_list(const ParameterList& param_list,
         proxy.availableBuiltinEndpoints =
             param.builtin_endpoints();
         break;
+      //TLS_TODO: verify it's the right place where secure parameters should be added to the proxy
```

Comment:
Are there other options?

Reply:
`ParticipantBuiltinTopicData` structure contains all secure members, and that's where I added extended_builtin_endpoints. On the other hand `ParticipantProxy_t` contains `BuiltinEndpointSet_t availableBuiltinEndpoints`, and that's where new `DDS::Security::ExtendedBuiltinEndpointSet_t availableExtendedBuiltinEndpoints' seems to belong too.

Reply:
So maybe a better question is whether it is useful to have `ParticipantBuiltinTopicData` storing `ExtendedBuiltinEndpointSet_t`.

Reply:
It's coming from the security spec, so keeping it with those makes sense.  We can always change it later.